edgexfoundry/app-functions-sdk-go
# build: Update to Go 1.20, Alpine 3.17 and linter v1.51.2

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      <link to docs PR>

## Testing Instructions
Run non-secure EdgeX stack with Device Virtual
Build and run app template from command line
Verify logs shows data flowing from Device Virtual
Verify /ping endpoint works

## New Dependency Instructions (If applicable)
